sql/sqlwire/sql_api.proto

```diff
@@ -29,9 +29,11 @@ option (gogoproto.unmarshaler_all) = true;
 
 // SQLRequestHeader is supplied with every CmdRequest.
 message SQLRequestHeader {
+  // User is the originating user.
+  optional string user = 5 [(gogoproto.nullable) = false];
   // Session settings that were returned in the last response that
   // contained them, being reflected back to the server.
-  optional bytes session = 1;
+  optional bytes settings = 1;
```

Comment:
I like the name `session` better than `settings`. `settings` implies to me that it's under the caller's full control; `session` sounds like a better match for the cookie-like echoing done here.